sessionctx/variable/varsutil.go

```diff
@@ -106,6 +106,10 @@ func GetSessionOnlySysVars(s *SessionVars, key string) (string, bool, error) {
 		return strconv.FormatUint(atomic.LoadUint64(&config.GetGlobalConfig().Log.SlowThreshold), 10), true, nil
 	case TiDBQueryLogMaxLen:
 		return strconv.FormatUint(atomic.LoadUint64(&config.GetGlobalConfig().Log.QueryLogMaxLen), 10), true, nil
+	case TiDBDDLReorgWorkerCount:
```

Comment:
Why put it here? The variable isn't a session variable.

Reply:
Yes. But this function is defined as `get the default value defined in code for session only variable`.